_posts/2020-02-10-q1-2020-update.md

```diff
@@ -0,0 +1,186 @@
+
+### Development
+
+**Budget: 29 000 USD**
+
+**Lead: ripcurlx**
+
+As a software project, development is a core function.
+
+Bisq shipped lots of significant updates to its software in 2019. Now, the focus is on refining this software to be more reliable and welcoming.
+
+For reliability, solving critical bugs are now a top development priority.
+
+We define "critical bug" to be any issue that:
+
+1. prevents a user from successfully trading or managing funds on Bisq
+2. repeatedly lands users in support
+3. puts user funds or privacy at risk
+
+We'll track progress on improvements by tracking metrics such as _critical bug incidents per DAO cycle_ and _critical bug incidents per trade_. Known critical bugs will be prominently tracked and displayed, and solutions to such bugs will get the highest priority in the development budget.
+
+For onboarding, we need to make Bisq less imposing for new users. Making your first trade on Bisq is not trivial, and many would-be Bisq users are turned away from the apparent complexity of getting started with it.
+
+Luckily, we already have some excellent mockups for significant improvements. These mockups include a more intuitive interface for the desktop software and (most crucially) a beautiful step-by-step wizard to help new users get started without guesswork.
+
+Moving forward, implementing these new interfaces will be a top priority.
+
+### Growth
+
+**Budget: 10 700 USD**
+
+**Lead: m52go**
+
+Growth efforts were intentionally muted through much of 2019 because of the DAO launch, bank account scammers, and other factors that made it unsuitable to attract new users to Bisq. The v1.0 and v1.2 releases solved many of these issues.
+
+As it stands at the beginning of 2020, the Bisq software isn't perfect, but it's ready for more users and more volume. We're looking to hit and sustain 2,100 BTC of trading volume per month as soon as possible. Although this is a big improvement from Bisq's current monthly volume, there were months in 2019 in which Bisq trading volume far exceeded 2,100 BTC.
+
+We plan to achieve this higher, more consistent trading volume with a smarter, more consistent approach to market-making and community engagement:
+
+1. Develop and promote strategic market-making bounties. The USD, EUR, and XMR markets are strong on Bisq, but volume fluctuates and liquidity is not consistent. We will incentivize consistent, desirable liquidity by making sure Bisq always has offers key markets that people want.
+2. Develop and nurture stronger ties with community. Bisq already has a good amount of word-of-mouth among those in its key demographics across a range of channels, but it needs to more proactively insert itself into more conversations and give more people more reasons to use Bisq.
```

Comment:
"with community" needs an article, e.g. "the" or something else to qualify. Maybe "Develop and nurture stronger ties with target audiences". I don't love that either, though.